pkgs/by-name/at/atlauncher/package.nix

```
@@ -1,50 +1,63 @@
-{ copyDesktopItems, fetchurl, jre, lib, makeDesktopItem, makeWrapper, stdenv, udev, xorg }:
+{ copyDesktopItems
+, fetchFromGitHub
+, fetchurl
+, lib
+, makeDesktopItem
+, makeWrapper
+, stdenv
```

stdenv pulls in a C compiler which can be unnecessary for times like these
```suggestion
, stdenvNoCC
```